<launch>
  <!-- ARMOR servce -->
  <node pkg="armor"
        type="execute"
        name="armor_service"
        args="it.emarolab.armor.ARMORMainService"
  />

  <!-- behaviour node -->
  <node pkg="exprob_assignment1"
        type="behaviour.py"
        name="behaviour"
        output="screen"
        launch-prefix="xterm -e"
  />

  <!-- robot_state node -->
  <node pkg="exprob_assignment1"
        type="robot_state.py"
        name="robot_state"
        output="screen"
        launch-prefix="xterm -e"
  />

  <!-- owl_interface node -->
  <node pkg="exprob_assignment1"
        type="owl_interface.py"
        name="owl_interface"
        output="screen"
        launch-prefix="xterm -e"
  />

  <!-- planner node -->
  <node pkg="exprob_assignment1"
        type="planner.py"
        name="planner"
        output="screen"
        launch-prefix="xterm -e"
  />

  <!-- controller node -->
  <node pkg="exprob_assignment1"
        type="controller.py"
        name="controller"
        output="screen"
        launch-prefix="xterm -e"
  />

  <!-- camera_manager node -->
  <node pkg="exprob_assignment1"
        type="camera_manager.py"
        name="camera_manager"
        output="screen"
        launch-prefix="xterm -e"
  />

</launch>
